docs/DeveloperGuide.md

```diff
@@ -257,29 +257,37 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
+* tech savy university students
 * prefer desktop apps over other types
-* can type fast
-* prefers typing to mouse interactions
-* is reasonably comfortable using CLI apps
+* can type fast and have plenty of experience with CLI application
+* is often busy and struggle to manage important details about their relationships
```

Comment:
A minor grammar nitpick, should it be 'has plenty of ...CLI applications', and 'struggles to manage ...'?